gdal/frmts/postgisraster/postgisrasterdataset.cpp

```diff
@@ -3021,13 +3026,14 @@ GetConnection(const char * pszFilename, char ** ppszConnectionString,
     OutDBResolution* peOutDBResolution)
 {
     PGconn * poConn = nullptr;
+    char * pszService = nullptr;
```

Comment:
should be CPFree 'd() at the end of the function

Reply:
Sure. Sorry for that.